### PR TITLE
[ui] Allow both horizontal and vertical renderings of the asset graph

### PR DESCRIPTION
**Pull request by bengotow**
## Summary & Motivation

Back by popular demand! The vertical asset graph returns as an option accessible from a context menu on the asset graph, a small button in the bottom right, and a shortcut (Alt-O)

image

image

image

This PR also makes some incremental + related improvements:

- A few icons have been resized to our standard 16x16 so the toolbar on the asset graph looks consistent.

- The icons shown in Menu are now textDefault instead of accentGray (Verified this change with Josh)

- I accidentally started putting the new context menu option in `AssetNodeContextMenu` before I realized I wanted it on the background context menu instead. In the process, I made a few code improvements there (shared type for the props, etc) that I left in the PR. 

- The edges on the graph now go OVER the group background but UNDER the group title bars, which helps a lot in vertical mode.

- The borders around asset groups are a color that uses opacity, so that the "stack lines" beneath them darken when they collide with edges.

- The op graph has switched back to the correct line styling. It was using horizontal joiners incorrectly.

Before:
image

After: 
image

## How I Tested These Changes

I referenced the storybooks for the asset node tweaks, but the rest of this I tested manually with a couple large graphs.

### CODE REVIEW

File: js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNodeMenu.tsx [hunk 1]
```diff
@@ -101,13 +76,17 @@ export const useAssetNodeMenu = ({
...
+            icon="arrow_back"
+            onClick={() => showGraph(`*\"${tokenForAssetKey(node.assetKey)}\"`)}
+          />
         ) : null}
         {downstream.length ? (
           <MenuItem
             text="Show downstream graph"
             icon="arrow_forward"
-            onClick={showDownstreamGraph}
+            onClick={() => showGraph(`\"${tokenForAssetKey(node.assetKey)}\"*`)}
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNodeMenu.tsx:89]: nice simplificaiton

**Review by salazarm — APPROVED** [review]
Sweet!

**Comment by salazarm** [comment]
I kind of want to right align the shortcuts in the context menu:
image

**Comment by salazarm** [comment]
Also I realize collapse/expand have the same shortcut 🤔  I wonder which one takes precedence if both are possible. Maybe we need different shortcuts or some way of indicating which would take precedence?

File: js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeLineageGraph.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const AssetNodeLineageGraph = ({
 
   // Use the pathname as part of the key so that different deployments don't invalidate each other's cached layout
```
Comment (salazarm) [inline js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeLineageGraph.tsx:45]: I think I forgot to delete this comment

Comment (bengotow) [reply js_modules/dagster-ui/packages/ui-core/src/assets/AssetNodeLineageGraph.tsx:45]: Ahh ok will delete it!

**Comment by bengotow** [comment]
Looks like in the mixed state you mentioned the shortcut performs "collapse all". I updated the code for the menu so the rendering reflects that:

image

**Pull request merged**